src/RA_Defs.cpp

```diff
@@ -10,68 +10,150 @@ GetParseErrorFunc GetJSONParseErrorStr = GetParseError_En;
 #endif
 
 static_assert(sizeof(BYTE*) == sizeof(char*), "dangerous cast ahead");
+
+_Use_decl_annotations_
```

Comment:
Don't know if you guys know what this is but it's for SAL (structured annotation language). It's if the declaration and definition are in separate locations.